### PR TITLE
remove unused cc expiry fields from sf query

### PR DESCRIPTION
**Pull request by pvighi**
removing everything but the signed off fields from the cc Expiry tab in :
https://docs.google.com/spreadsheets/d/1pGC_HHE6Ba91j4CEpidtAJPRB3ceIepuWgpuHT-KhQY/edit#gid=243144275

### CODE REVIEW

File: handlers/sf-datalake-export/src/main/scala/com/gu/sf_datalake_export/salesforce_bulk_api/SfQueries.scala [hunk 1]
```diff
@@ -238,49 +238,20 @@ object SfQueries {
...
-      |Initial_Payment_Method_Created_By__c,
-      |Latest_Payment_Method_Created_By__c,
-      |Initial_Payment_Method_Name__c,
-      |Latest_Payment_Method_Name__c,
-      |Payment_Term__c,
-      |Product_Name__c,
-      |Product_Type__c,
-      |Promotion_Code__c,
-      |Rate_Plan_Name__c,
-      |Id,
```
Comment (paulbrown1982) [inline handlers/sf-datalake-export/src/main/scala/com/gu/sf_datalake_export/salesforce_bulk_api/SfQueries.scala]: I think we might need this, or the CreatedDate in order to dedupe sequential card expiries during a customer's lifecycle.

Comment (paulbrown1982) [reply handlers/sf-datalake-export/src/main/scala/com/gu/sf_datalake_export/salesforce_bulk_api/SfQueries.scala]: Still can't decide whether we want the CreatedDate too? What if we want to plot a series of cardexpiries over time, for example...

Comment (pvighi) [reply handlers/sf-datalake-export/src/main/scala/com/gu/sf_datalake_export/salesforce_bulk_api/SfQueries.scala]: I added it, we can remove it later if we decide we do not want it

**Comment by paulbrown1982** [comment]
👍

**Pull request merged**